festim/temperature/temperature_solver.py

```diff
@@ -36,15 +36,15 @@ class HeatTransferProblem(festim.Temperature):
     def __init__(
         self,
         transient=True,
-        initial_value=0.0,
+        initial_condition=None,
```

Comment:
Among other thing. I'm not sure what `initial_condition` should be set as default, if `transient=True`.

Reply:
@RemDelaporteMathurin, both suggestions are included in the latest commit